src/compiler/typescript_to_rust/generators/azle_into_js_value_trait.ts

```diff
@@ -64,6 +64,83 @@ export function generateAzleIntoJsValueTrait(): Rust {
             }
         }
 
+        impl AzleIntoJsValue for ic_cdk::api::call::RejectionCode {
```

Comment:
We might want to write a Vec impl for ic_cdk::api::call::RejectionCode as well just in case and since all of the other types have Vec impls